server/app.py

```diff
@@ -293,7 +293,19 @@ async def construct_and_send(input: AvatarInput, websocket: WebSocket, conversat
     )
 
     # send the speech to websocket
```

Comment:
This is bad because if model changes, we need to update this. Instead do the following

message_pydantic = await MessagePydantic.from_tortoise_orm(message)
    # send the speech to websocket
    await websocket.send_json(message_pydantic.model_dump())